src/valueObjects/Collection.js

```diff
@@ -23,7 +43,7 @@ class FolderCollection {
   }
 
   entrySlug(path) {
-    return path.split('/').pop().replace(/\.[^\.]+$/, '');
+    return slugFormatter(this.collection.get('slug'), path);
   }
```

Comment:
This breaks things badly! We should only use the `slugFormatter` to generate new slugs! When we actually have a `path` the slug will always be the last part of the path without the extension.